### ISSUE TITLE
Keyboard trapped in Mozilla Demo

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/content#103]
When <kbd>Tab</kbd>bing through http://localhost:5000/en-US/docs/Web/HTML/Element/input/text I can't leave the iFrame of Mozilla Demo using <kbd>Tab</kbd>.

Instead the key is used inside the HTML editor there.

**Comment by bershanskiy** [comment]
@Ryuno-Ki I agree that people should be able to navigate outside of the editor via a key press (for convenience and accessibility). However, I believe it's the expected behavior that Tab key creates a Tab when pressed within the editor. Perhaps, there should be a key combination that behaves like regular Tab when pressed within the editor?

**Comment by Ryuno-Ki** [comment]
Hm, looking through fitting WCAG or ARIA examples, I think, it would be [application role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Application_Role).

For this, I could imagine leaving the demo by hitting `<Esc>` and offering a way to jump over it just before the Mozilla Demo is injected could be a good option.

**Comment by Malvoz** [comment]
Is it important to have tabs within the editor? If not the editor could simply be a [`contenteditable`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable) element?

**Comment by sideshowbarker** [comment]
Closing, as it seems this became moot after the move to Yari and away from the wiki system

**Comment by Malvoz** [comment]
Or should I open a new issue, in this repo or elsewhere?

**Comment by sideshowbarker** [comment]
> Did you misinterpret this issue perhaps?

Yes, I did. Re-opening and moving to the yari repo
